driveruipanels.cpp

```diff
@@ -558,6 +558,19 @@ debug_panel::render() {
                     m_input.mover->AlarmChainFlag = false;
```

Comment:
That whole existing "Fix status" section is just wrong.
Simulation data can't be ever modified outside of command processing calls.

"Fix status" button should only call something like
```
relay command_relay;
relay.post(user_command::resetconsist, 0.0, 0.0, GLFW_PRESS, 0, glm::vec3(0.0f), &m_input.vehicle->name());
```
and pantograph reset code should be added to resetconsist handler inside simulation.cpp